src/Microsoft.IdentityModel.Protocols/AuthenticationProtocolMessage.cs

```diff
@@ -86,7 +86,7 @@ public virtual string BuildFormPost()
         /// Builds a Url using the current IssuerAddress and the parameters that have been set.
         /// </summary>
         /// <returns>UrlEncoded string.</returns>
-        /// <remarks>Each parameter &lt;Key, Value> is first transformed using <see cref="HttpUtility.UrlEncode(string)"/>.</remarks>
+        /// <remarks>Each parameter &lt;Key, Value> is first transformed using <see cref="Uri.EscapeDataString(string)"/>.</remarks>
```

Comment:
The opening < is HTML encoded, but the closing one isn't. Should they both be encoded or should they both not be encoded?